app/classifier/tasks/drawing/summary.cjsx

```diff
@@ -1,4 +1,6 @@
 React = require 'react'
+strip = require 'strip-markdown'
+remark = (require 'remark').use(strip)
```

Comment:
Agreed. I don't think we should add a dependency just for this. Could we consider just rendering the markdown in the summary, too?